scratch/PCA/extract_data.py

```diff
@@ -0,0 +1,123 @@
+
+# coding: utf-8
+
+# In[61]:
+
+#!/usr/bin/env python
+
+"""
+DESCRIPTION
+
+    Script for scraping process parameters and defect parameters for PCA
+    Outfitted for use with 'process_parameters.csv'
```

Comment:
It would be better to have an intermediate function between the input file format and the input to the function. This ensures that if data is read from another source, this function remains unchanged.